packages/server/src/utils/constants.ts

```diff
@@ -48,6 +50,7 @@ export const PORT = ENVVARS.PORT;
 export const ROOT_URL = ENVVARS.ROOT_URL;
 export const SELLER_APP_URL = ENVVARS.SELLER_APP_URL;
 export const SELLER_STORYBOOK_URL = ENVVARS.SELLER_STORYBOOK_URL;
+export const USE_REMOTE_DEV_DB = ENVVARS.USE_REMOTE_DEV_DB;
```

Comment:
what do you think about converting to a boolean here?
```suggestion
export const USE_REMOTE_DEV_DB = ENVVARS.USE_REMOTE_DEV_DB === 'true';
```

Reply:
I've been trying with this for a while, and for some reason I can't get our `ENVVARS` values to be typed as `string | boolean`, they really wanna be strings. Also that change is a bit involved and can probably be its own PR as a chore.
I'm fine with leaving it as a `string` for now and refactoring later because we have similar code like this for one of the other env vars as well. What do you think?

Reply:
https://app.clickup.com/t/dwpak3

Reply:
Okay so hear this right. I swear I tried this out and it didn't work.
But then I tried it again and it works :-)

Reply:
Will update🤣

Reply:
updated in 70a084731b93ae650b9b862bb9d208ceec4d9479